.changeset/gentle-oranges-crash.md

```diff
@@ -0,0 +1,24 @@
+---
+'@openfn/language-salesforce': major
+---
+
+New API design for salesforce, including adding composeNextState and removing
+old code.
+
+### Major Changes
+
+- Remove axios dependency
+- Remove old/unused functions. `relationship`, `upsertIf`, `createIf`,
```

Comment:
There's no migration guide for upsertIf and createIf, which I think is worth capturing

Reply:
I have updated the changeset to include a migration guide

Reply:
It looks to be duplicated?

Reply:
My bad 🙌🏽 , i have cleaned up the duplicates